lib/vanagon/platform/windows.rb

```diff
@@ -107,21 +107,22 @@ def generate_msi_package(project)
         target_dir = project.repo ? output_dir(project.repo) : output_dir
         cg_name = "ProductComponentGroup"
         dir_ref = "INSTALLDIR"
+        candle_flags =  "-dPlatform=#{@architecture} -arch #{@architecture} -ext WiXUtilExtension -ext WixUIExtension"
```

Comment:
@melissa As discussed, I think the best approach here is to ticket a separate task to ensure that the version payload is included correctly in the MSI.

I think the existing bill-of-materials code is also missing component version numbers, so will note that in the ticket.